mono/dis/util.c

```diff
@@ -20,9 +20,8 @@
 #endif
 
 /**
- * map:
- * @code: code to lookup in table
- * @table: table to decode code
+ * \param code code to lookup in table
```

Comment:
It's ok, ask around.
We can't lose a way to publish the existing docs on the embedding API alone (no internals).